### ISSUE TITLE
Loudly Proclaim the Solution!

### ISSUE THREAD

**Issue by mlissner** [DmitriK/darkContrast#46]
I've been using this add-on for a while and have generally appreciated it. Today though, I finally learned about a much better solution. In `about:config`, you can set:

    widget.content.gtk-theme-override

To a light them, like `Adwaita:light`, and it will render widgets using that theme! 

I think this is big enough news that you should consider doing a popup tab for anybody that uses this extension on Linux. I think if you add something like this to your background page, it should work when the user gets the upgrade:

```
function showNotificationTab(details){
  // Show some kind of notification tab to the user after install/upgrade.
  console.debug("showing install/upgrade notification if version matches");
  let currentVersion = chrome.runtime.getManifest().version;
  if (details.reason === 'update' && currentVersion === '1.2.3'){
    // This version is when we pushed for using about:config. Show that page.
    chrome.tabs.create({
      url: 'https://some-url-you-control'
    });
  }
}

chrome.runtime.onInstalled.addListener(showNotificationTab);
```

On top of this, I think you should put a big notice about this in the configuration page so people notice it there, and in the description of the add-on, so people notice it there, before they install. 

In short: I've loved this extension for quite a while, but I think you should do everything in your power to make people use about:config and to uninstall this extension.

**Comment by DmitriK** [comment]
Added some information about this to the readme. I'll still maintain this extension for users who wish to keep some native styles in content. I'd rather not use a popup as that is a bit too invasive IMO.

**Comment by mlissner** [comment]
It's super invasive, but I think a lot of people would be happy to see it an uninstall an extension.

**Comment by xrat** [comment]
`widget.content.gtk-theme-override` has a different effect. I much prefer darkContrast! The override allows to specify a "light" theme (dark text color on light background) to fix the problem of illegible text in widgets. darkContrast, however, 

> does not modify element colors if it determines that they already have good contrast.

For instance, I have `<textarea>`s with no colors defined, which, hence, use the theme's colors. With the override I get dark text on light background. With the add-on I get my theme's colors which is light gray on black.
